v2.1/cockroach-commands.md

```diff
@@ -19,7 +19,7 @@ Command | Usage
 [`cockroach quit`](stop-a-node.html) | Temporarily stop a node or permanently remove a node.
 [`cockroach sql`](use-the-built-in-sql-client.html) | Use the built-in SQL client.
 [`cockroach user`](create-and-manage-users.html) | Get, set, list, and remove users.
-[`cockroach zone`](configure-replication-zones.html) | Configure the number and location of replicas for specific sets of data.
+[`cockroach zone`](configure-replication-zones.html) | _Deprecated._ To configure the number and location of replicas for specific sets of data, use [`ALTER ... CONFIGURE ZONE`](configure-zone.html).
```

Comment:
Do other things use italics for this?  Per our [Style guide](https://github.com/cockroachdb/docs/blob/master/STYLE.md) I think this should be bold.  Also, in this case I would argue for bold anyway so that the deprecation status stands out more visually.